project1/import.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from flask import Flask, session
+from flask_session import Session
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
+from flask import render_template, request, session
+from books import *
+import csv
+
+app1 = Flask(__name__)
+
+# Check for environment variable
+# if not os.getenv("DATABASE_URL"):
+#     raise RuntimeError("DATABASE_URL is not set")
+
+# Configure session to use filesystem
+app1.config["SESSION_PERMANENT"] = False
+app1.config["SESSION_TYPE"] = "filesystem"
+Session(app1)
+
+app1.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app1.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app1.app_context().push()
+
+db1.init_app(app1)
+db1.create_all()
```

Comment:
let all the tables be created at once, no need of separately writing again for creating books table.

Reply:
> let all the tables be created at once, no need of separately writing again for creating books table.


Hi Ma'am,

Thanks for the review. Actually I feel better keeping a new py file for books. To make myself free from confusion, I discussed with my team and created a seperate file claaed books.py.
If you consider my reason is valid, please close the request. Kindly let me know for any further modification.

Regards

Reply:
I mean tables can be created from application.py itself as you have the same code over there, get the instance of db in books.py to insert

Reply:
Hi Ma'am, Thanks for the suggestion.
I have removed the retundat code and made a single file for creating databases. 
removed addiotnal python file an checked the code is working fine now.